### DISCUSSION TITLE
Get page number in .docx file

### DISCUSSION THREAD

**Discussion by FrenchGithubUser** [jgm/pandoc#9535] [Q&A]
I am using pandoc to conver a `docx` file to `md` which works fine. But I'd like to associate every heading to a page number for further processing. Is there a way to do so ? Or maybe a way to convert the `docx` file to another format which will be more convenient to extract the page numbers from ?

What I currently tried to do was to convert the `docx` file to `pdf` and then match the headings from the markdown to the ones in the pdf. But this works poorly as some headings are not formatted in the same way in the pdf/docx.

**Comment by jgm** [comment]
No, I don't think there's any way to do this with pandoc.  Page numbers are not available in the docx itself; they are constructed by Word in rendering it.

**Reply by FrenchGithubUser** [reply]
Yea I know this is how it's done, but maybe there was a way for pandoc to do the same calculation that MS word does

**Reply by jgm** [reply]
To do that we'd have to implement Word's page layout algorithm.  Not really feasible.

**Reply by FrenchGithubUser** [reply]
I see... But libreoffice has some open source libs that might help to do that ? However I don't know if the calculations are really the same in between libreoffic and MS word
